VespaGAS/Models/Gasoline.swift

```diff
@@ -1 +1,11 @@
-typealias Gasoline = Volume
+import Foundation
+
+struct Gasoline: Volume {
+    var amount: Float
```

Comment:
I believe these could be `let` - think it only has to be `var` in the protocol declaration